src/golive/checklist.md

```diff
@@ -15,14 +15,33 @@ You have a domain name registered for your site with a Registrar of your choice.
 
-## 3. Optionally obtain a 3rd party SSL certificate
+If your production environment is on a Dedicated instance, ensure that the code is up to date in both your `staging` and `production` branches, as those are what will be mirrored to the Dedicated instances.  Also ensure that the data on the production instance is up to date and ready to launch.
 
-Platform.sh automatically provides SSL certificates for all sites issued by [Let's Encrypt](https://letsencrypt.org/) at no charge.  In most cases this is sufficient and no further action is necessary.  However, if you want to use a 3rd party SSL certificate to encrypt your production site you can obtain one from any number of 3rd party SSL issuers.  Platform.sh does not charge for using a 3rd party SSL certificate, although the issuer may.
+## 3. Optionally obtain a 3rd party TLS certificate
+
+Platform.sh automatically provides TLS certificates for all sites issued by [Let's Encrypt](https://letsencrypt.org/) at no charge.  In most cases this is sufficient and no further action is necessary.  However, if you want to use a 3rd party TLS certificate to encrypt your production site you can obtain one from any number of 3rd party TLS issuers.  Platform.sh does not charge for using a 3rd party TLS certificate, although the issuer may.
 
 Platform.sh supports all kinds of certificates including domain-validated certificates, extended validation (EV) certificates, high-assurance certificates and wildcard certificates.  The use of HA or EV certificates is the main reason why you may wish to use a third party issuer rather than the default certificate.  You will also need a custom certificate if you use wildcard routes, as Let's Encrypt does not support wildcard certificates.
 
 If you do wish to use a 3rd party certificate, ensure it is purchased and active prior to going live.
 
+## 4. Optionally configure your CDN
+
+If you are using a CDN, either one included with an Enterprise plan or one you provide for a self-service Grid project, ensure that your CDN account is registered and configured in advance.  That includes setting the upstream on your CDN to point to the Platform.sh production instance.  
+
+* For a Grid-based project, that will be the `master-xxxx` domain.  Run `platform environment:info edge_hostname` to get the domain name to use.
+* For a Dedicated project, the upstream to use will be provided by your Platform.sh onboarding representative.
+
+Consult your CDN's documentation for how to set the CDN's upstream address.
+
+For Enterprise plans you may need to obtain a DNS TXT record from your Platform.sh support representative by opening a ticket.  Consult the documentation for your CDN provider and our own [CDN guide](/golive/cdn.md).
```

Comment:
I may be getting confused here on the Enterprise plan vs. Dedicated project terminology...